service_crategen/Cargo.toml

```diff
@@ -31,6 +31,12 @@ env_logger = "0.4.0"
 optional = true
 version = "0.0"
 
+[profile.dev]
+opt-level = 1
```

Comment:
On my Macbook these settings are almost an even tradeoff: compilation takes longer but the runtime is greatly reduced.  The scales will shift to shortened runtime being more important as we add the ~20 services we're missing.